src/rf/apps/core/models.py

```diff
@@ -230,10 +243,6 @@ class LayerImage(Model):
         max_length=255,
         help_text='Filename of original file',
     )
-    s3_uuid = UUIDField(
```

Comment:
the uuid field has nice properties. First, it is a native type in postgres which means that lookups are faster than on a pure text field. Also, it has a unique constraint on it to avoid possible collisions. I chose it for these two reasons as I felt with many images the lookups might start to slow down and it seemed a better unique constraint than a varchar field.